content/akeneo-web-api.yaml

```diff
@@ -2462,6 +2476,15 @@ responses:
       "code": 415,
       "message": "‘xxx’ in ‘Content-type’ header is not valid.  Only ‘application/vnd.akeneo.collection+json’ is allowed."
     }
+  415ErrorMultipart:
+    description: Unsupported Media type
+    x-details: The `Content-type` header is not `multipart/form-data` but it should
```

Comment:
I've just modified `application/json` into `multipart/form-data`. The other sentences (for `application/json` and `application/vnd.akeneo.collection+json` are the same. But I can modify it (both sentences are ok for me).